### DISCUSSION TITLE
Multiple RPCs?

### DISCUSSION THREAD

**Discussion by SammBrooks** [FluxpointDev/MultiRPC#40] [Old]
Does this actually allow for multiple RPCs to be active at once?
Say having an RPC for both PlexRPC (music) and AdobeRPC (photo editing) at once in the same status?

I found this while searching for a possible solution for this particular need.

**Comment by xXBuilderBXx** [comment]
Discord already displays multiple activities but the user popup will only show 1 (not sure on the order)
You can also use this with different Discord accounts by using the 3 different clients Discord, Discord PTB and Discord Canary.

**Reply by SammBrooks** [reply]
Yeah, I know that it does already. I was just curious if there was a way for them to stack or be in a "show more" menu or some sort of merger that puts them into one widget.
Thanks for the info though.

**Reply by xXBuilderBXx** [reply]
No Discord only shows 1 on profile unless you click the activities tab, there is no merge.

**Comment by xXBuilderBXx** [comment]
No Discord only shows 1 on profile unless you click the activities tab, there is no merge.

**Comment by xXBuilderBXx** [comment]
Redoing MultiRPC
